src/auction.js

```diff
@@ -386,6 +386,10 @@ export function doCallbacksIfTimedout(auctionInstance, bidResponse) {
 
 // Add a bid to the auction.
 export function addBidToAuction(auctionInstance, bidResponse) {
+  let bidderRequests = auctionInstance.getBidRequests();
+  let bidderRequest = find(bidderRequests, bidderRequest => bidderRequest.bidderCode === bidResponse.bidderCode);
+  setupBidTargeting(bidResponse, bidderRequest);
```

Comment:
Why are we calling this function for all formats ? We need `hb_uuid` and `hb_cache_id` only when cache is done i.e only for video as of now. Won't it be easier if we attach after hook function to `callPrebidCache`. That way these two targeting keys will only be added if we do caching.